### ISSUE TITLE
MathML validation assistance

### ISSUE THREAD

**Issue by adamsilverstein** [ampproject/amphtml#12973]
## What's the issue?

I'm having trouble getting the new amp-mathml component to validate properly on https://github.com/ampproject/amphtml/pull/12836 

Passing validation is a blocker for merge and I would appreciate some help as I am new to contributing here and don't quite understand what is causing the validation failure. On the PR @honeybadgerdontcare suggested I create a new issue and assign to them, thanks again for all the help!

**Comment by honeybadgerdontcare** [comment]
File:
`extensions/amp-mathml/0.1/test/amp-mathml.html`

Contents:
```
<!doctype html>
<html ⚡>
<head>
  <meta charset="utf-8">
  <title>amp-mathml example</title>
  <link rel="canonical" href="amps.html">
  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
  <style amp-custom>
    amp-mathml {
      color: red;
    }
  </style>
  <script async custom-element="amp-mathml" src="https://cdn.ampproject.org/v0/amp-mathml-0.1.js"></script>
  <script async src="https://cdn.ampproject.org/v0.js"></script>
</head>
<body>
  <h2>The Quadratic Formula</h2>
  <amp-mathml layout="container" data-formula="\[x = {-b \pm \sqrt{b^2-4ac} \over 2a}.\]">
  </amp-mathml>
  <h2>Cauchy's Integral Formula</h2>
  <amp-mathml layout="container" data-formula="\[f(a) = \frac{1}{2\pi i} \oint\frac{f(z)}{z-a}dz\]">
  </amp-mathml>
  <h2>Double angle formula for Cosines</h2>
  <amp-mathml layout="container" data-formula="\[ \cos(θ+φ)=\cos(θ)\cos(φ)−\sin(θ)\sin(φ) \]">
  </amp-mathml>
  <h2>Inline formula.</h2>
  This is an example of a formula placed  inline in the middle of a block of text. <amp-mathml layout="container" inline data-formula="\[ \cos(θ+φ) \]"></amp-mathml> This shows how the formula will fit inside a block of text and can be styled with CSS.
</body>
</html>
```

**Comment by Gregable** [comment]
Happy to help. A few questions:

1. The extension spec you committed referred to the extension as `amp-html`. I assume you mean `amp-mathml`?
2. The extension spec you committed uses the attribute `formula` and here you use `data-formula`. Which matches the implementation you had in mind?

**Comment by honeybadgerdontcare** [comment]
Fixed in #13001
